services/brig/src/Brig/API/Public.hs

```diff
@@ -1065,10 +1067,43 @@ activate (Public.Activate tgt code dryrun)
 
 -- Verification
 
-sendVerificationCode :: (Handler r) ()
-sendVerificationCode =
-  case Public.TeamFeatureSndFPasswordChallengeNotImplemented of
-    _ -> pure ()
+sendVerificationCode :: Public.SendVerificationCode -> (Handler r) ()
+sendVerificationCode req = do
+  let action = Public.svcAction req
+  -- TODO(leif): guard as long as not all of the actions are implemented
+  -- remove when all done
+  case action of
+    Public.GenerateScimToken -> pure ()
+    Public.Login -> do
+      let email = Public.svcEmail req
+      mbUserId <- lift $ UserKey.lookupKey $ UserKey.userEmailKey email
+      mbTeamId <- lift $ join <$> Intra.getTeamId `traverse` mbUserId
+      mbStatus <- lift $ Intra.getTeamSndFactorPasswordChallenge `traverse` mbTeamId
+      let featureStatus = maybe (Public.tfwoapsStatus Public.defaultTeamFeatureSndFactorPasswordChallengeStatus) Public.tfwoStatus mbStatus
+      case (mbUserId, featureStatus) of
+        (Just userId, Public.TeamFeatureEnabled) -> do
+          gen <- Code.mk6DigitGen $ Code.ForEmail email
+          mbPendingCode <- lift $ Code.lookup (Code.genKey gen) (scope action)
+          case mbPendingCode of
+            Nothing -> do
+              code <-
+                Code.generate
+                  gen
+                  (scope action)
+                  (Code.Retries 3)
+                  (Code.Timeout (10 * 60)) -- 10 minutes
```

Comment:
this needs to be configurable in brig.yaml.  can we use [`setActivationTimeout`](https://github.com/wireapp/wire-server/blob/8281461864f623d78d78ef82a27390a5a06de49b/services/brig/src/Brig/Options.hs#L424)?